### ISSUE TITLE
.mume directory is created in home directory not in .atom directory

### ISSUE THREAD

**Issue by RickTorresJr** [shd101wyy/mume#66]
.mume directory is created in home directory not in .atom directory

**Comment by shd101wyy** [comment]
Hi @RickTorresJr , 

There is also a [vscode-markdown-preview-enhanced](https://github.com/shd101wyy/vscode-markdown-preview-enhanced) package for Visual Studio Code. Both the Atom and VSCode versions of MPE share the same `~/.mume` directory.

Thank you

**Comment by Tb0ne-root** [comment]
Can we remove it?

**Comment by ispanos** [comment]
@shd101wyy 

This directory should be in `$XDG_CONFIG_HOME`, or default to `.config/mume` if the environmental variable is not set. 

https://wiki.archlinux.org/index.php/XDG_Base_Directory

**Comment by SebDanielsson** [comment]
Yes, please stop dumping config files in our home directories. Having all config in .config directroy would be much appreciated😊
